hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -343,22 +595,209 @@
          :read (assoc op :type :ok, :value (into (sorted-set) (.get m "hi")))))
 
      (teardown! [this test]
-       (.terminate conn)))))
+       (.shutdown conn)))))
 
 (defn map-workload
   "A workload for map tests, with the given client options."
   [client-opts]
-  {:client    (map-client client-opts)
-   :generator (->> (range)
-                   (map (fn [x] {:type  :invoke
-                                 :f     :add
-                                 :value x}))
-                   gen/seq
-                   (gen/stagger 1/10))
+  {:client          (map-client client-opts)
+   :generator       (->> (range)
+                         (map (fn [x] {:type  :invoke
+                                       :f     :add
+                                       :value x}))
+                         gen/seq
+                         (gen/stagger 1/10))
    :final-generator (->> {:type :invoke, :f :read}
                          gen/once
                          gen/each)
-   :checker   (checker/set)})
+   :checker         (checker/set)})
+
+
+
+(defn parseLine [line]
+  (let
+    [tokens (.split line CLIENT_ID_SEPARATOR)]
+    [(:value (clojure.edn/read-string (nth tokens 2))) (.trim (nth tokens 1))]))
+
+(def clientUidsToClientNames (memoize (fn []
+                            (apply array-map
+                                   (flatten
+                                     (map parseLine
+                                          (.split (:out (sh "grep" CLIENT_ID_SEPARATOR "store/latest/jepsen.log")) "\n")
```

Comment:
I strongly recommend against doing this. Store your client ID mappings in the test map instead.